test_fit.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+#include <string.h>
+
+static void placeCharacter(char *string, unsigned int sizeOfcharacter)
```

Comment:
```suggestion
static void PlaceNullCharacter(char *string, unsigned int sizeOfCharacter)
```

Reply:
This doesn't just place any character, it places a null-terminating character so we need to make sure that the name makes this clear.